### ISSUE TITLE
Terraform version detection broken for >0.9.0?

### ISSUE THREAD

**Issue by seeingidog2** [sepulworld/tfsoa#34]
Looks like this is broken and we're assuming it's an old version. Will need to look into it.

<img width="311" alt="screen shot 2017-06-21 at 4 43 36 pm" src="https://user-images.githubusercontent.com/26415029/27411178-f5218510-56a0-11e7-819f-c4a068bd6077.png">

**Comment by sepulworld** [comment]
Good call. Yeah, I think we need to pull the state down to push to API. In 0.9.0 and newer I think Terraform stopped keeping a copy of the state local
